packages/core/upload/admin/src/components/MediaLibraryInput/Carousel/CarouselAssetActions.js

```diff
@@ -43,7 +43,7 @@ export const CarouselAssetActions = ({ asset, onDeleteAsset, onAddAsset, onEditA
       {onEditAsset && (
         <IconButton
           label={formatMessage({
-            id: getTrad('app.utils.edit'),
+            id: 'app.utils.edit',
```

Comment:
good catch!
I think we should keep the `getTrad` and use the `control-card.edit` translation key though 👀

Reply:
sure, updated https://github.com/strapi/strapi/pull/15261/commits/9ed95854ccaf7aac3a0714f7249484cc3b8279ba